mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -365,7 +365,7 @@ class LogisticRegression @Since("1.2.0") (
       case None => histogram.length
     }
 
-    val isMultinomial = $(family) match {
+    val isMultinomial = $(family).toLowerCase match {
```

Comment:
It can, but I think it would need to be done in the concrete `setXXX` method each time.